src/main/java/org/snomed/snowstorm/core/data/services/CodeSystemUpgradeService.java

```diff
@@ -50,24 +57,89 @@ public class CodeSystemUpgradeService {
 	@Value("${snowstorm.rest-api.readonly}")
 	private boolean isReadOnly;
 
+	private static final Map<String, CodeSystemUpgradeJob> upgradeJobMap = new HashMap<>();
+
 	private final Logger logger = LoggerFactory.getLogger(getClass());
 
+	public CodeSystemUpgradeService() {
+		crunchifyTimer = new Timer();
```

Comment:
Why call it a crunchifyTimer here please @QuyenLy87 ?   Does that mean something?  It's just that the object created is just a Timer, so it's like you're saying something extra, but I don't know what crunchify means?